config/symbols.us.stno0.txt

```diff
@@ -62,11 +62,16 @@ UpdateStageEntities = 0x801C3AB4;
 HitDetection = 0x801C3BBC;
 EntityDamageDisplay = 0x801C4CD4;
 CreateEntityFromLayout = 0x801C5378;
-CreateEntityWhenInHorizontalRange = 0x801C543C;
-FindFirstEntityAbove = 0x801C566C;
-FindFirstEntityBelow = 0x801C56B8;
-CreateEntitiesAbove = 0x801C5710;
-CreateEntitiesBelow = 0x801C580C;
```

Comment:
The auto symbol mapping had a hard time matching these up - think @hohle noticed a similar thing on stream with AxeKnightDeath / MakeExplosions.

This is because Above / Right and Below / Left are very close matches so it will just map the first one it finds I think.

Reply:
These are similar enough I think you would just need special hardcoded rules in the tooling